evaluation/mint/run_infer.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+export PYTHONPATH=$(pwd)
+# poetry shell
+
+python ./evaluation/mint/run_infer.py \
```

Comment:
Can you refer to swe_bench's run_infer.sh to add the AgentVersion to `eval_note`?